onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -289,6 +293,14 @@ static bool ParseDimensionOverride(std::basic_string<ORTCHAR_T>& dim_identifier,
         }
         break;
       }
+      case 'k': {
```

Comment:
For consistency we should remove the curly braces (other switch cases don't have them, except when required for initializing local variables).